stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -557,7 +596,7 @@ private fun DeletedMessageFooter(
 
         Timestamp(
             modifier = Modifier.padding(8.dp),
-            date = message.updatedAt ?: message.createdAt
+            date = message.updatedAt ?: message.createdAt ?: Date()
```

Comment:
Do we really need this `Date()` here?

Reply:
In this case I put it in yeah - because when we upload images, the message _exists_ but doesn't really. There is no `updatedAt` or `createdAt` date, but we need to show a timestamp.

It's a weird case, which is why I put it just in here, instead of in the `Timestamp` for example.